akka-bbb-apps/src/main/scala/org/bigbluebutton/endpoint/redis/RedisRecorderActor.scala

```diff
@@ -141,6 +141,10 @@ class RedisRecorderActor(
       ev.setMeetingId(msg.header.meetingId)
       ev.setSenderId(msg.body.msg.sender.id)
       ev.setMessage(msg.body.msg.message)
+      ev.setSenderRole(msg.body.msg.sender.role)
+      
+      val isModerator = msg.body.sender.role == "MODERATOR"
```

Comment:
```suggestion
      val isModerator = msg.body.msg.sender.role == "MODERATOR"
```
I was the one who suggested this but I didn't notice it was missing a `msg` after the `body` prop. My bad!